### PR TITLE
V2 General Fixes 

### PR DESCRIPTION
**Pull request by sean-brydon**
Various fixes in the v2 space

Fixes some of the issues mentioned in https://github.com/calcom/cal.com/issues/4009

Booking Page

CleanShot 2022-08-30 at 15 30 04

Button Group & Switch Fix - Removed embed button for now as the implementation needs changed to reflect the new tabName prop being used within the layout 

CleanShot 2022-08-30 at 15 31 01

Also fixes the hover state on active datepicker

### CODE REVIEW

File: apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx [hunk 1]
```diff
@@ -191,7 +191,8 @@ function EventTypeSingleLayout({
                 showToast("Link copied!", "success");
               }}
             />
-            <Button color="secondary" size="icon" StartIcon={Icon.FiCode} combined />
+            {/* TODO: Implement embed here @hariom */}
```
Comment (sean-brydon) [inline apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx:194]: @hariombalhara Did you ever get a chance to look at the tabName conflict?

File: packages/ui/v2/core/Button.tsx [hunk 2]
```diff
@@ -84,10 +84,12 @@ export const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, ButtonPr
         // base styles independent what type of button it is
         "inline-flex justify-center items-center text-sm font-medium relative",
         // different styles depending on size
-        size === "base" && "h-9 px-4 py-2.5 rounded-md ",
-        size === "lg" && "h-[36px] px-4 py-2.5 rounded-md",
-        size === "icon" && "flex justify-center h-[36px] w-[36px] rounded-md",
-        combined && "rounded-none first:border-r-0 last:border-l-0 first:rounded-l-md last:rounded-r-md ",
+        size === "base" && "h-9 px-4 py-2.5  ",
```
Comment (sean-brydon) [inline packages/ui/v2/core/Button.tsx:87]: Fix silly active hover when in light mode

Comment (sean-brydon) [inline packages/ui/v2/core/Button.tsx:87]: Fixes combined to work as intended

File: packages/ui/v2/core/layouts/BookingLayout.tsx [hunk 3]
```diff
@@ -11,10 +11,16 @@ const tabs: VerticalTabItemProps[] = [
     href: "/bookings/upcoming",
     icon: Icon.FiCalendar,
   },
+  // TODO: Add filter for unconfimred bookings in a future PR - Out of scope
```
Comment (sean-brydon) [inline packages/ui/v2/core/layouts/BookingLayout.tsx:14]: This filter doesn't work on the backend so was throwing errors - Will create a issue

**Review by PeerRich — APPROVED** [review]
great stuff!

**Comment by PeerRich** [comment]
NIT: the icon is missing

now:
CleanShot 2022-08-30 at 22 04 02@2x


figma:
CleanShot 2022-08-30 at 22 04 24@2x

**Review by pumfleet — APPROVED** [review]

File: packages/ui/v2/core/Button.tsx [hunk 4]
```diff
@@ -43,13 +43,16 @@ const variantClassName = {
   secondary: "border border-gray-200 text-brand-900 bg-white hover:bg-gray-100",
   minimal:
     "text-gray-700 bg-transparent hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-offset-1 focus:bg-gray-100 focus:ring-brand-900 dark:text-darkgray-900 hover:dark:text-gray-50",
+  minimalSecondary:
```
Comment (sean-brydon) [inline packages/ui/v2/core/Button.tsx:46]: Created a new variant for minimal + outline hover - as we have a few designs that use both minimal/minimalSecondary

**Pull request merged**